gulpfile.js

```diff
@@ -19,78 +14,87 @@ plugins.uglify = require('gulp-uglify');
-      }),
-      rollupPluginCommonjs({
-        include: 'node_modules/**',
-      }),
-    ],
-    output: {
-      format: 'iife',
-      name: 'GOVUK',
-    },
-  })
-    .pipe(source('all.mjs'))
-    .pipe(buffer())
-    .pipe(
-      plugins.addSrc.prepend([
-        paths.npm + 'jquery/dist/jquery.min.js',
-        paths.npm + 'query-command-supported/dist/queryCommandSupported.min.js',
-        paths.npm + 'textarea-caret/index.js',
-        paths.npm + 'cbor-js/cbor.js',
-        paths.npm + 'd3/dist/d3.min.js',
-        paths.npm + 'socket.io-client/dist/socket.io.min.js'
-      ])
-    );
-
+  // Files that don't use NotifyModules and can be uglified
   const local = src([
-    paths.toolkit + 'javascripts/govuk/modules.js',
-    paths.toolkit + 'javascripts/govuk/show-hide-content.js',
-    paths.src + 'javascripts/copyToClipboard.js',
-    paths.src + 'javascripts/enhancedTextbox.js',
-    paths.src + 'javascripts/fileUpload.js',
+    paths.src + 'javascripts/modules/init.js',
+    paths.src + 'javascripts/modules/uswds-modules.js',
+    paths.src + 'javascripts/modules/show-hide-content.js',
     paths.src + 'javascripts/radioSelect.js',
-    paths.src + 'javascripts/listEntry.js',
     paths.src + 'javascripts/liveSearch.js',
-    paths.src + 'javascripts/errorTracking.js',
     paths.src + 'javascripts/preventDuplicateFormSubmissions.js',
-    paths.src + 'javascripts/fullscreenTable.js',
-    paths.src + 'javascripts/templateFolderForm.js',
-    paths.src + 'javascripts/collapsibleCheckboxes.js',
-    paths.src + 'javascripts/radioSlider.js',
-    paths.src + 'javascripts/updateStatus.js',
     paths.src + 'javascripts/errorBanner.js',
     paths.src + 'javascripts/notifyModal.js',
     paths.src + 'javascripts/timeoutPopup.js',
     paths.src + 'javascripts/date.js',
     paths.src + 'javascripts/loginAlert.js',
-    paths.src + 'javascripts/main.js',
-    paths.src + 'javascripts/totalMessagesChart.js',
-    paths.src + 'javascripts/activityChart.js',
     paths.src + 'javascripts/sidenav.js',
     paths.src + 'javascripts/validation.js',
-    paths.src + 'javascripts/job-polling.js',
     paths.src + 'javascripts/scrollPosition.js',
   ])
+
+  // Files that use NotifyModules - split into two groups to avoid stream issues
+  const notifyModules1 = src([
+    paths.src + 'javascripts/copyToClipboard.js',
+    paths.src + 'javascripts/enhancedTextbox.js',
+    paths.src + 'javascripts/fileUpload.js',
+    paths.src + 'javascripts/errorTracking.js',
+    paths.src + 'javascripts/fullscreenTable.js',
+    paths.src + 'javascripts/templateFolderForm.js',
+  ])
+    .pipe(plugins.prettyerror())
+    .pipe(
+      plugins.babel({
+        presets: ['@babel/preset-env'],
+      })
+    );
+
+  const notifyModules2 = src([
+    paths.src + 'javascripts/collapsibleCheckboxes.js',
+    paths.src + 'javascripts/radioSlider.js',
+    paths.src + 'javascripts/updateStatus.js',
+    paths.src + 'javascripts/main.js',
+  ])
```

Comment:
Asking out of curiosity - what led to these needing to be split into separate groups like this?